docs/specs/openapi.yaml

```diff
@@ -183,6 +426,41 @@ components:
               type: array
               items:
                 $ref: "#/components/schemas/serviceInfo"
+    PostServices:
+      allOf:
+        - $ref: '#/components/schemas/BaseResponse'
+        - type: object
+          properties:
+            change:
+              type: string
+              description: The Change ID of the asynchronous change.
+            result:
+              type:
+                - object
```

Comment:
When can `POST /v1/services` return an object for `result`? Looking at `v1PostServices()` in api_services.go, I only see `return AsyncResponse(nil, change.ID())` for returning a result.

Reply:
Yes, you are absolutely right, after checking the code, the only valid value for "result" is null. There is a case where the result is not empty when there is an error but it's irrelevant here.